tests/test_restaurants.py

```diff
@@ -0,0 +1,21 @@
+def test_get_all(test_client, get_mocker):
+    data = [{"address": "Minsk", "id": 0, "name": "Vasilki", "phone_number": "+375297777777",
```

Comment:
this data is the same as in `get_mocker`, it is possible to avoid-copy paste:

1. create a fixture with your data:
   ```python
   @pytest.fixture()
   def restaurants():
     return [{"address": "Minsk", "id": 0, "name": "Vasilki", "phone_number": "+375297777777",
                                "work_time": "Monday-Sunday: 08:00 - 23:45"},
                               {"address": "Minsk", "id": 1, "name": "Mama Doma", "phone_number": "+375336666666",
                                "work_time": "Monday-Sunday: 10:00 - 22:00"}]
   ```

2. inject the fixture with data in both `test_get_all` and `get_mocker` like:
```python
def get_mocker(mocker, restaurants):
    yield restaurants

def test_get_all(test_client, get_mocker, restaurants):
  ...
```

Reply:
fixed